drivers/serial/Kconfig.uwp

```diff
@@ -0,0 +1,34 @@
+# UNISOC UWP UART configuration
+
+# Copyright (c) 2018, UNISOC Incorporated
+# SPDX-License-Identifier: Apache-2.0
+
+config UART_UWP
+	depends on SOC_SERIES_UWP566X
+	bool "UWP UART driver"
+	default n
+	select SERIAL_HAS_DRIVER
+	select SERIAL_SUPPORT_INTERRUPT
+	depends on SOC_FAMILY_UWP
+	help
+	  This option enables the UWP UART driver, for UART_0.
```

Comment:
Is this an instance config or driver config? We've recently removed driver instance Kconfigs, please see 5a149dfca697e2716d0e96770be1358c9ec0e618